x-pack/test/case_api_integration/security_and_spaces/tests/common/configure/get_connectors.ts

```diff
@@ -8,86 +8,18 @@
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../../../common/ftr_provider_context';
 
-import { ObjectRemover as ActionsRemover } from '../../../../../alerting_api_integration/common/lib';
-import {
-  getCaseConnectors,
-  createConnector,
-  getServiceNowConnector,
-  getJiraConnector,
-  getResilientConnector,
-  getServiceNowSIRConnector,
-  getWebhookConnector,
-} from '../../../../common/lib/utils';
+import { getCaseConnectors } from '../../../../common/lib/utils';
 
 // eslint-disable-next-line import/no-default-export
 export default ({ getService }: FtrProviderContext): void => {
   const supertest = getService('supertest');
-  const actionsRemover = new ActionsRemover(supertest);
 
   describe('get_connectors', () => {
-    afterEach(async () => {
-      await actionsRemover.removeAll();
-    });
-
     it('should return an empty find body correctly if no connectors are loaded', async () => {
       const connectors = await getCaseConnectors(supertest);
       expect(connectors).to.eql([]);
     });
 
-    it('should return case owned connectors', async () => {
```

Comment:
This test would fail with a basic license. It mostly already existed in a trial test but I added the service now connector.